kytos/core/auth.py

```diff
@@ -44,6 +50,83 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class UserController:
```

Comment:
@Alopalao, we'll need to have retries for `AutoReconnect` recoverable error that `pymongo` can raise since IO failures can happen here, let's add the [`for_all_method(retries, ...)`](https://github.com/kytos-ng/flow_manager/blob/master/controllers/__init__.py#L24-L35) decorator like flow_manager has been using

Reply:
Oh that is what it does. Added.